modules/nf-core/antismash/antismashlitedownloaddatabases/main.nf

```diff
@@ -62,7 +63,7 @@ process ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        antismash-lite: \$(antismash --version | sed 's/antiSMASH //')
+        antismash-lite: $VERSION
```

Comment:
I saw the discussion about this issue on Slack, so it sounds like this is good to go!